scripts/build.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+set -e
+
+# Make sure submodule is registered and up-to-date.
+git submodule update --init
+
+# Apply the patch
+cd ckeditor-dev
+git checkout master
+git pull
+git checkout liferay
+git reset --hard origin/master
+git am ../patches/*
```

Comment:
Maybe something to prepare people for failure:

```sh
echo "Applying patches from \"patches/\" directory."
if ! git am ../patches/*; then
  echo "There was a problem applying patches:"
  echo
  echo "To retry manually and fix:"
  echo
  echo "  cd ckeditor-dev"
  echo "  git am --abort"
  echo "  git am ../patches/*"
  echo
  echo "Once you are happy with the result, run scripts/patch.sh to update the contents of \"patches/\"."
  exit 1
fi
```

Reply:
Yes, sounds like a better option